docs/modules/default-architecture/pages/loading.adoc

```diff
@@ -0,0 +1,188 @@
+= Loading
+
+Start with LangChain for loading your data instead of manually coding your own pipeline.
+The chances are good that LangChain already provides the functionalities you need.
+
+The examples use Astra {db-vector} database for the vector store and assume you have one available. If not, see xref:ROOT:prerequisites.adoc[].
+
+== Load PDF from file
+
+. Use the PyPDFLoader to load PDFs as an array of `Document` objects.
+Each `Document` object contains the page content and metadata - in this case, the page number.
++
+[source,python]
+----
+from langchain.document_loaders import PyPDFLoader
```

Comment:
similar comment on imports for this and all import sections on this page


 https://api.python.langchain.com/en/stable/document_loaders/langchain_community.document_loaders.pdf.PyPDFLoader.html?highlight=pypdfloader#langchain_community.document_loaders.pdf.PyPDFLoader